tox.ini

```diff
@@ -0,0 +1,43 @@
+[tox]
+envlist = py27,py3{6,7,8,9,10,11},flake8,docs
+skip_missing_interpreters = true
+isolated_build = true
+skipsdist = true
+
+[flake8]
+max-line-length = 120
+
+[testenv]
+skip_install = true
+install_command = pip install {opts} {packages}
+
+[testenv:flake8]
+basepython = python3
+commands =
+    flake8 ssg utils
+deps =
+    flake8
+
+[testenv:docs]
+skip_install=false
+skipsdist=true
+basepython = python3
+allowlist_externals =
+    make
+deps =
+    -r docs/requirements.txt
+commands =
+    make -C docs html
+    # make -C docs linkcheck
```

Comment:
Mainly because both commands produce a ton of console output, and focusing on just one seemed to make sense.  I can drop the linkcheck or make a separate tox cmd, or something else?

Reply:
I just re-enabled that ^^ as a post-command. Let me know if I should leave it that way.